rs/rosetta-api/icrc1/ledger/sm-tests/src/in_memory_ledger.rs

```diff
@@ -69,6 +70,7 @@ where
     pub allowances: HashMap<K, Allowance<Tokens>>,
     pub total_supply: Tokens,
     pub fee_collector: Option<AccountId>,
+    pub burns_without_spender: Option<BurnsWithoutSpender<AccountId>>,
```

Comment:
Does this variable (and above) need to be public? In another PR Nikolas mentioned we use such practice, but I am not sure what is the benefit? I am not against making it public if needed, but what is the benefit of using pub by default?

Reply:
Good point; I also prefer to have fields, methods etc. private if it's not necessary to have them public. I changed these to private.